site/src/pages/RoadmapPage/Year.tsx

```diff
@@ -81,6 +80,24 @@ const Year: FC<YearProps> = ({ yearIndex, data }) => {
                 <Button variant="light" className="year-settings-btn">
                   Edit Year
                 </Button>
+                <Button 
+                  variant="light" 
+                  className="year-settings-btn"
+                  id="clear-btn"
+                  onClick={() => {
+                    for (let i = 0; i < data.quarters.length; i++) {
```

Comment:
Same thing here. I would create an action that takes in a year index and sets all the quarters to an empty array. No need for double for loops. And abstracts the logic into the redux slice.